packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,38 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value = 0,
+  max = 100,
+  size = 'regular',
+  hideLabel,
+  children,
+  className,
+}: ProgressProps): JSX.Element => {
+  const normalizedValue = Math.round((value / max) * 100);
+
+  return (
+    <div
+      className={classy(c('progress'), m(size), className)}
```

Comment:
We usually spread the props first, then override what's needed.

```suggestion
    <div
      {...restProps}
      className={classy(c('progress'), m(size), className)}
```

Reply:
Done!